applications/tasks/src/modules/kt_compactor_worker.erl

```diff
@@ -288,12 +296,13 @@ min_data_met(_Data, _Min) ->
 
 -spec min_ratio_met(integer(), integer(), float()) -> boolean().
 min_ratio_met(Disk, Data, MinRatio) ->
-    case Disk / Data of
+    case (Disk - Data) / Disk * 100 of
```

Comment:
More info about this formula can be found here http://docs.couchdb.org/en/stable/config/compaction.html#compaction-daemon-rules